crates/rpc/rpc/src/admin.rs

```diff
@@ -37,7 +37,7 @@ where
 {
     /// Handler for `admin_addPeer`
     fn add_peer(&self, record: NodeRecord) -> RpcResult<bool> {
-        self.network.add_peer(record.id, record.tcp_addr());
+        self.network.add_peer(record.id, record.tcp_addr(), Some(record.udp_addr()));
```

Comment:
is this the actual fix?

Reply:
actually no. I think this one is one of the the fix: 

https://github.com/paradigmxyz/reth/pull/9043/files#diff-baf630ae3977373c999a197ca26e45fe2d8453b4e16090dbcd3efdc95b70398fR607-R613